iroh-sync/src/heads.rs

```diff
@@ -0,0 +1,53 @@
+//! Author heads
+
+use std::collections::BTreeMap;
+
+use serde::{Deserialize, Serialize};
+
+use crate::AuthorId;
+
+/// Timestamps of the latest entry for each author.
+// TODO: If we want to encode this in gossip messages we have to enforce a max length.
```

Comment:
aren't these already sent on gossip messages?

Reply:
WDYM *these* here?

Reply:
The "this" in the "TODO" 😅

Reply:
Fixed in the latest commits. We now only encode the `n` most recent `(AuthorId, Timestamp)` pairs until we reach the gossip max message size.